test/Microsoft.Azure.WebJobs.ServiceBus.UnitTests/MessageToStringConverterTests.cs

```diff
@@ -50,9 +50,11 @@ public class MessageToStringConverterTests
         [InlineData(ContentTypes.ApplicationOctetStream, TestString)]
         [InlineData(null, TestJson)]
         [InlineData("application/xml", TestJson)]
+        [InlineData(ContentTypes.TextPlain, null)]
+        [InlineData(ContentTypes.TextPlain, "")]
         public async Task ConvertAsync_ReturnsExpectedResult_WithSerializedString(string contentType, string value)
         {
-            Message message = new Message(Encoding.UTF8.GetBytes(value));
+            Message message = new Message(value == null ? null : Encoding.UTF8.GetBytes(value));
```

Comment:
Add similar tests for null and empty in ConvertAsync_ReturnsExpectedResult_WithBinarySerializer test

Reply:
Added only null, DataContractBinarySerializer converts empty string to @string3http://schemas.microsoft.com/2003/10/Serialization/